source/_components/homematic.markdown

```diff
@@ -83,26 +83,40 @@ We use three approaches to fetch the names of devices. Each assumes you have pro
 
 Resolving names can take some time. So when you start Home Assistant you won't see you devices at first. For a setup with 20+ devices it can take up to a minute until all devices show up in the UI.
 
-### Multiple hosts
+### {% linkable_title Multiple hosts %}
 
 In order to allow communication with multiple hosts or different protocols in parallel (wireless, wired and ip), multiple connections will be established, each to the configured destination. The name you choose for the host has to be unique and limited to ASCII letters.  
 Using multiple hosts has the drawback, that the services (explained below) may not work as expected. Only one connection can be used for services, which limits the devices/variables a service can use to the scope/protocol of the host.  
 This does *not* affect the entites in Home Assistant. They all use their own connection and work as expected.
 
-### Variables
+### {% linkable_title Reading attributes of entities %}
 
-It is possible to read and set values of system variables you have setup on the CCU/Homegear. An example of how that is done can be found below. The supported types for setting values are float- and bool-variables.  
-Each variable will be available as it's own entity in the form of `homematic.name`. The predefined `homematic.homematic` variable has the number of service messages as it's value. You can use these variable-entities like any other entity in Home Assistant to trigger automations.  
+Most devices have, besides their state, additional attributes like their battery state or valve position. These can be accessed using templates in automations, or even as their own entities using the [template sensor](https://home-assistant.io/components/sensor.template/) component. Here's an example of a template sensor that exposes the valve state of a thermostat.
+
+```yaml
+sensor:
+- platform: template
+  sensors:
+    bedroom_valve:
+      value_template: '{{ states.climate.leq123456.attributes.Valve }}'
```

Comment:
To avoid render of the template, use `{% raw %}{{ your_template }}{% endraw %}`.